lib/models/BucketInfo.js

```diff
@@ -97,6 +100,11 @@ class BucketInfo {
         }
         if (locationConstraint) {
             assert.strictEqual(typeof locationConstraint, 'string');
+            ingestion = locationConstraint.split(zenkoSeparator)[1];
+        }
+        if (ingestionConfig) {
+            ingestion = ingestionConfig;
+            assert.strictEqual(typeof ingestion, 'string');
```

Comment:
when will `locationConstraint` not have the ingestion flag but will have `ingestionConfig`?

Reply:
the ingestion flag would be its own separate value on a bucketInfo model, `locationConstraint` will be the carrier of the ingestion flag but will not store the ingestion value